src/Components/Signup.jsx

```diff
@@ -17,6 +18,7 @@ class Signup extends Component {
     name: '',
     city: '',
     country: 'Sweden',
+    role: 'user',
```

Comment:
why do you set the role in state if you cant change it?

Reply:
changed it to empty string :)